### PR TITLE
balancer: do some cleanup.

### PR DESCRIPTION
**Pull request by huachaohuang**
Do some cleanup.

### CODE REVIEW

File: server/balancer.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ type Balancer interface {
 	ScoreType() scoreType
 }
 
-func selectFromStore(stores []*storeInfo, excluded map[uint64]struct{}, filters []Filter, st scoreType) *storeInfo {
+func selectFromStore(stores []*storeInfo, excluded map[uint64]bool, filters []Filter, st scoreType) *storeInfo {
```
Comment (siddontang) [inline server/balancer.go]: I prefer map[uint64]struct{}
@qiuyesuifeng ?

File: conf/config.toml [hunk 2]
```diff
@@ -18,7 +18,7 @@ initial-cluster-state = "new"
 
 
 lease = 3
-log-level = "debug"
+log-level = "info"
```
Comment (qiuyesuifeng) [inline conf/config.toml]: I think we should also update the log level flag value to `info`.

File: server/balancer.go (hunk 1 above)
Comment (qiuyesuifeng) [inline server/balancer.go]: Seems no difference,  but usually we use map[uint64]struct{} instead of map[uint64]bool, it is more effective.

Comment (huachaohuang) [inline server/balancer.go]: OK, I don't much about it, it just looks weird to me :)

File: conf/config.toml (hunk 2 above)
Comment (huachaohuang) [inline conf/config.toml]: It is default to `info`.

**Comment by siddontang** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**